extension/json/src/json_scan.cpp

```diff
@@ -8,25 +12,62 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace json_extension {
 
-struct JsonBindData : public TableFuncBindData {
+struct JsonScanConfig {
+    std::string format;
+    int64_t depth;
+    int64_t breadth;
+
+    explicit JsonScanConfig(const std::unordered_map<std::string, Value>& options)
+        : format("array"), depth(-1), breadth(-1) {
```

Comment:
instead of giving default value there, i would do the following:
std::string format = DEFAULT_JSON_FORMAT
    int64_t depth = DEFAULT_JSON_DEPTH
    int64_t breadth = DEFAULT_JSON_BREADTH
    and declare those three variables as constants